instance/models/openedx_appserver.py

```diff
@@ -23,6 +23,7 @@
 from django.db import models
 from django.template import loader
 from django.utils.text import slugify
+from django_extensions.db.fields.json import JSONField
```

Comment:
Can we not use the official one instead? `from django.contrib.postgres.fields import JSONField`